Charter integration

hlepori opened the pull request:
This is a proposed update that aims to integrate the FIXM charter into the User Manual, so that its content becomes visible & searchable, as discussed within the FIXM CCB. The proposed update includes the integration of the technology "mermaid", which enables users to create web-based diagrams and interact with them very easily, and allows for simplified maintenance and integration of the diagrams that currently exist in the charter. 

The proposed update also integrates an initial attempt to address the visibility concerns for strategic documents such as the charter if these documents are moved to the User Manual.

This pull request relates to the following FIXM CCB actions:
Action 17: EUROCONTROL/Hubert: Integrate the charter content with the web-based user manual and promote to testing environment.

hlepori committed:
Add Mermaid plugin

hlepori committed:
Create charter.md

hlepori committed:
Create strategy.md

hlepori committed:
Create versioning.md

hlepori committed:
Create releasePlan.md

hlepori committed:
Update home.md

Added overview of FIXM Strategic Documents

hlepori committed:
Update _sidebar.md

Added links to Strategic FIXM Documents

hlepori committed:
Update home.md

Add links to mermaid diagram

cmcgowan-CTR reviewed (approved):
I was originally looking for the Charter Management document content only so I was a little confused at first but then realized you have also incorporated the FIXM Strategy and FIXM Version Plan content as well. Looks good. 
Assuming the final resting place for this content will be further discussed at Novembers CCB.

hlepori commented:
> I was originally looking for the Charter Management document content only so I was a little confused at first but then realized you have also incorporated the FIXM Strategy and FIXM Version Plan content as well. Looks good. Assuming the final resting place for this content will be further discussed at Novembers CCB.

Thank you for the feedback, Caroline. Indeed, I incorporated the FIXM Strategy and (a subset of) the FIXM Release Plan content as well to help the FIXM CCB evaluate whether the visibility of these strategic “documents” remains satisfactory if promoted to docs.fixm.aero. Nick had rightly pointed out that this visibility question should be tackled and evaluated. Certainly, the visibility of these document should NOT be undermined. Everything to be further discussed by the FIXM CCB of course.

CPiroli-CTR reviewed (approved):
Hi all.  It's a little hard to visualize what these changes will look like until they are merged (or is there some way to do this and I missed it?) but, as long as they faithfully represent the content of the documents, I think they will be fine.  I do question whether or not the User Manual is the appropriate place for these docs (the average FIXM user won't care about the organization of the CCB, for example) but agree that putting them online is a good step.  Perhaps just keeping them in a separate repository is the key?

hlepori commented:
> Hi all. It's a little hard to visualize what these changes will look like until they are merged (or is there some way to do this and I missed it?) but, as long as they faithfully represent the content of the documents, I think they will be fine. I do question whether or not the User Manual is the appropriate place for these docs (the average FIXM user won't care about the organization of the CCB, for example) but agree that putting them online is a good step. Perhaps just keeping them in a separate repository is the key?

Here are two screenshots showing how it would look like - I have used a personal workspace to do the initial testing.
![charter diagram](https://user-images.githubusercontent.com/70144023/141956732-2ab16c96-c94f-44ef-9f9e-24cba96ce778.png)
 
![main page](https://user-images.githubusercontent.com/70144023/141956763-b3c9492d-7304-46a3-9305-36e7caa8a907.png)

hlepori commented:
An alternative could be indeed to have another repository for storing these strategic documents. But what would be the url, since "docs.fixm.aero" is already used for the FIXM User Manual? Maybe "ccb.fixm.aero"?
